centos_ci.sh

```diff
@@ -305,7 +305,7 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags=basic,openshift,version,cmd-config,profile -format pretty"
+  MINISHIFT_VM_DRIVER=kvm make integration_all GODOG_OPTS="-format pretty"
```

Comment:
I don't see any need of passing ```GODOG_OPTS``` from here. This can be passed directly in ```integration_all``` target.

Reply:
@amitkrout , Not sure how to achieve that. I am getting error
```
$ make integration_all -format preety
make: ormat: No such file or directory
make: *** No rule to make target 'ormat'.  Stop.
```

Reply:
@budhrg pass it directly from integration_all 
```
go test -timeout $(TIMEOUT) $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) -tags ~coolstore -format pretty
```